test_todoapp.py

```diff
@@ -22,18 +32,20 @@ def client():
     with test_app.test_client() as client:
         yield client
 
-@mock.patch('requests.get')
+@patch('requests.get')
```

Comment:
This test is still patching requests, even though we're now using mongo -  we should reenable this test and fix it up